nise/__main__.py

```diff
@@ -243,7 +268,14 @@ def create_parser():
     yaml_parser = subparsers.add_parser("yaml", help="Generate a yaml for creating cost usage reports.")
 
     add_yaml_parser_args(yaml_parser)
-
+    report_parser.add_argument(
+        "-c",
+        "--currency",
+        dest="currency",
+        required=False,
+        type=valid_currency,
+        help="Sets the currency code on the reports",
```

Comment:
```suggestion
        help="Sets the currency code on the reports, this will also override any currency in static ymls",
```

Reply:
Just added and pushed this up thanks for the catch 😄